lib/src/main/java/uk/gov/di/ipv/cri/drivingpermit/library/domain/DrivingPermitForm.java

```diff
@@ -58,11 +58,11 @@ public DrivingPermitForm(
         this.licenceIssuer = licenceIssuer;
         this.drivingLicenceNumber = drivingLicenceNumber;
         this.postcode = postcode;
-        List<Address> addresses = new ArrayList<>();
+        List<Address> tempAddresses = new ArrayList<>();
         Address address = new Address();
         address.setPostalCode(postcode);
-        addresses.add(address);
-        this.addresses = addresses;
+        tempAddresses.add(address);
```

Comment:
Would it work to get rid of temp addresses doing
this.addresses = List.of(address)